.github/workflows/main.yml

```diff
@@ -0,0 +1,138 @@
+name: main
```

Comment:
Lets have a well defined workflow name

Reply:
We thought of having the summary page as main.html and thereby this name.
As per the other comment, we can change the file name to health-dashboard and make an update here as well.

Reply:
Updated